src/ZfcDatagrid/Column/AbstractColumn.php

```diff
@@ -602,19 +669,43 @@ public function getRendererParameters($rendererName = 'jqGrid')
     }
 
     /**
-     * Set a a template formatter
+     * Set a template formatter and overwrite other formatter
+     *
+     * @param AbstractFormatter|AbstractFormatter[] $formatter
+     * @return $this
+     */
+    public function setFormatter($formatterList)
```

Comment:
Did u leave `setFormatter` for BC compability?

I would suggest to create a `setFormatters()` for multiple and leave that method for only one and deprecate it